packages/shared/src/parsers.ts

```diff
@@ -84,6 +84,11 @@ export const ethereumAddress = hexString.refine(
   (address) => ({ message: `${address} is not a valid ethereum address` }),
 );
 
+export const chainflipAddress = string.refine(
+  (address) => address.startsWith('cF') && decodeAddress(address),
```

Comment:
I imagine the RPC will accept any substrate address but it doesn't hurt

Reply:
yeah i think allowing only formatted addresses is a bit safer